takwimu/models/dashboard.py

```diff
@@ -631,50 +647,131 @@ class ProfilePage(ModelMeta, Page):
 
     subpage_types = ['takwimu.ProfileSectionPage']
 
-    def get_absolute_url(self):
-        return self.full_url
+    def get_promotion_image(self):
+        if self.promotion_image:
+            return self.promotion_image.file.url
 
-    def get_context(self, request):
-        context = super(ProfilePage, self).get_context(request)
-        
+    def get_country(self):
+        slug = self.slug
         for code, names in COUNTRIES.items():
-            if self.slug == slugify(names['name']):
-                context['country'] = {
+            if slug == slugify(names['name']):
+                return {
                     'iso_code': code,
                     'name': names['name'],
                     'short_name': names['short_name'],
                     'slug': slugify(names['name'])
                 }
-                break
+        return {}
+
+    def get_absolute_url(self):
+        return self.full_url
+
+    def get_context(self, request):
+        context = super(ProfilePage, self).get_context(request)
 
         context['meta'] = self.as_meta(request)
+
         return context
 
-    def get_promotion_image(self):
-        if self.promotion_image:
-            return self.promotion_image.file.url
+
+class AboutTakwimuContentBlock(blocks.StructBlock):
+    label = blocks.CharBlock(default="About Takwimu", max_length=255,
+                             help_text="Short title used in navigation, etc."),
+    title = blocks.CharBlock(default="About Takwimu", max_length=1024)
+    description = blocks.RichTextBlock(required=False, default="<p>People need accurate, objective information to make good decisions. However, uneven access to quality information weakens the impact of policy and programming in Africa as well as the ability of local development actors, particularly those with limited resources, to drive necessary change.</p><br/><p>Takwimu was launched in 2018 to empower African changemakers with the best data available and support their efforts to put this into effective use. We take a holistic view of what kinds of data are needed to drive change.</p><br/><p>Takwimu provides access to a growing body of national and sub-national statistics in the health, agriculture, education and financial inclusion sectors - combined with expert analysis of the key stakeholders, decision processes, policies, organisations and budgets that are driving development outcomes. All Takwimu content is visualised and packaged to be easily understood and freely shared.</p><br/><p>Takwimu currently covers 10 countries: Burkina Faso, Democratic Republic of Congo, Ethiopia, Kenya, Nigeria, Senegal, South Africa, Tanzania, Uganda and Zambia, in English and French. We&rsquo;re working to add more countries and additional languages. Takwimu is supported by the Bill &amp; Melinda Gates Foundation</p>")
```

Comment:
@kilemensi maybe move the default description and make it a constant outside the class, makes for cleaner code

Reply:
Done.